src/Tools.pm

```diff
@@ -401,31 +412,41 @@ DMRaid Devices look like:
 DMRaid Partitions look like:
 <strange name>_part\d
 
-
 =cut
 
-sub ReadDMRaidPartitions {
+# FIXME: this has to be done through Storage
```

Comment:
BTW actually requirement( and one of feature I implement) for JEOS is to depend only on `perl-base` and not whole `perl`